app/Http/Controllers/Bible/TextController.php

```diff
@@ -86,87 +86,19 @@ public function index($bible_url_param = null, $book_url_param = null, $chapter_
         }
         $asset_id = $fileset->asset_id;
         $cache_params = [$asset_id, $fileset_id, $book_id, $chapter, $verse_start, $verse_end];
-        $verses = cacheRemember('bible_text', $cache_params, now()->addDay(), function () use ($fileset, $bible, $book, $chapter, $verse_start, $verse_end) {
```

Comment:
I was expecting that TextController would be reverted back to the earlier version. The updated comments on line 50/51 are ok, but I don't understand why this big changes was made